CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 ## Unreleased
+
+### Build Fixes
+
+- **Apple Silicon**: Adds support to build and run tests on M1 Macs
+
```

Comment:
this might not be necessary as the changelog is automatically updated by the release process

Reply:
I could take it out.